errai-bom/pom.xml

```diff
@@ -76,22 +76,17 @@
     <jboss.web.version>7.0.16.Final</jboss.web.version>
     <jsr305.version>1.3.9</jsr305.version>
     <mojo.executor.version>2.2.0</mojo.executor.version>
-    <netty.codec.http.version>4.1.59.Final</netty.codec.http.version>
+    <netty.codec.http.version>4.1.48.Final</netty.codec.http.version>
```

Comment:
I think we should keep this upgrade or do 4.1.63 as it is in Wildfly https://github.com/wildfly/wildfly/blob/23.x/pom.xml#L304
This was originally done due https://github.com/netty/netty/security/advisories/GHSA-5mcr-gq6c-3hq2, so we should not revert this version